src/security/PeerOptions.cc

```diff
@@ -755,6 +759,7 @@ Security::PeerOptions::updateContextTrust(Security::ContextPointer &ctx)
 #elif USE_GNUTLS
     // Modern GnuTLS versions trust intermediate CA certificates by default.
 #endif /* TLS library */
+    (void)ctx;
```

Comment:
This would disable a useful warning if a buggy `USE_OPENSSL` code modification drops `ctx`. I think we should clone this line into `USE_GNUTLS` and add an `#else` use case.

Reply:
This change request was marked as resolved, but the problem is still there AFAICT.

Reply:
Fixed in commit 029035e67

Reply:
> Fixed in commit 029035e67

That commit missed the "and add an `#else` use case" part of the original change [request](https://github.com/squid-cache/squid/pull/667/files#r632118612).